openpower/configs/habanero_defconfig

```diff
@@ -64,3 +64,8 @@ BR2_OCC_BIN_FILENAME="occ.bin"
 BR2_CAPP_UCODE_BIN_FILENAME="cappucode.bin"
 BR2_PACKAGE_LOADKEYS=y
 BR2_IMA_CATALOG_DTS="POWER8"
+
+# Secureboot options
+BR2_OPENPOWER_SECUREBOOT_ENABLED=y
+BR2_OPENPOWER_SECUREBOOT_SIGN_MODE="development"
```

Comment:
Yeah, did this much just to prep for the remaining op-build patches that need to be checked in to activate the signing tool in building the PNOR.  It's true, at the moment, to build the sb-signing-utils project, only the first parm is required.  But as soon as it's activated, both will be required.  But I can removed it and let it be part of the other patches.